android/modules/platform/src/java/ti/modules/titanium/platform/DisplayCapsProxy.java

```diff
@@ -67,10 +67,20 @@ public String getDensity() {
 				return "high";
 			case DisplayMetrics.DENSITY_MEDIUM :
 				return "medium";
-			case 320 : // DisplayMetrics.DENSITY_XHIGH (API 9)
+			case 213: //TV
```

Comment:
Shouldn't this return 'tv' to be in line with http://developer.android.com/reference/android/util/DisplayMetrics.html?